### PR TITLE
Feature/36 hamburger menu

### PR DESCRIPTION
**Pull request by chris-lawton**
Adds nav and search toggle functionality

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/base.html [hunk 1]
```diff
@@ -38,38 +38,46 @@
...
+                    <div class="nav" data-nav>
+                        <p>Nav here</p>
+                        {% primarynav %}
+                        {% secondarynav %}
+                    </div>
 
-            <section class="header__menus header__menus--mobile" data-mobile-menu>
-                {% primarynav %}
+                    <div class="search" data-search>
+                        <p>Search here</p>
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/base.html]: Meant to be commited :P?

File: rca/project_styleguide/templates/patterns/base.html [hunk 2]
```diff
@@ -38,38 +38,46 @@
...
 
-                {% primarynav %}
-            </section>
+                        <a class="header__menu-close" href="#" data-close-menu>
+                            <svg width="14" height="14"><use xlink:href="#close"></use></svg>
+                        </a>
+                    </div>
+
+                    <div class="nav" data-nav>
+                        <p>Nav here</p>
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/base.html]: Is this a comment?

File: rca/static_src/sass/components/_nav.scss [hunk 3]
```diff
@@ -1,12 +1,24 @@
 .nav {
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-    width: 100%;
-    height: 100%;
+    // temp comment out
```
Comment (nicklee) [inline rca/static_src/sass/components/_nav.scss]: Did you mean to commit the comments here and below?

File: rca/static_src/sass/components/_search.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.search {
+    display: none;
+
+    .search-active & {
+        display: block;
+    }
+
+    p {
+        color: $color--white; //temp
```
Comment (nicklee) [inline rca/static_src/sass/components/_search.scss]: // Temp :D?

File: rca/static_src/sass/layout/_header.scss [hunk 5]
```diff
@@ -26,7 +26,80 @@
     }
 
     &__menu-toggle {
-        margin-right: ($gutter);
+        margin-top: $grid / 4;
```
Comment (nicklee) [inline rca/static_src/sass/layout/_header.scss]: I've previously been told to wrap all calculations in () for easier reading, so $grid / 4 would become ($grid / 4), don't know if you want to follow this convention? Would make sense for us to be consistent. 

I'm happy either way, just as long as are both doing the same.

File: rca/static_src/sass/layout/_header.scss [hunk 6]
```diff
@@ -26,7 +26,80 @@
...
+        &--medium {
+            display: none;
+
+            @include media-query(medium) {
+                display: flex;
+            }
+        }
+
+        .app__navigation & {
+            margin-top: $grid * 2.5;
```
Comment (nicklee) [inline rca/static_src/sass/layout/_header.scss]: Same as above ()

File: rca/static_src/sass/layout/_header.scss [hunk 7]
```diff
@@ -26,7 +26,80 @@
...
+            opacity: 0.4;
+
+            @include media-query(medium) {
+                margin-top: $grid / 4;
```
Comment (nicklee) [inline rca/static_src/sass/layout/_header.scss]: Same as above ()

File: rca/project_styleguide/templates/patterns/base.html (hunk 1 above)
Comment (chris-lawton) [reply rca/project_styleguide/templates/patterns/base.html]: left it in for testing by DM however they can probable see the difference in toggle opacity so will remove

File: rca/static_src/sass/layout/_header.scss (hunk 5 above)
Comment (chris-lawton) [reply rca/static_src/sass/layout/_header.scss]: can do 👍

**Pull request merged**